jobsdb/unionQuery.go

```diff
@@ -39,7 +39,7 @@ type MultiTenantJobsDB interface {
 	GetPileUpCounts(map[string]map[string]int)
 }
 
-func (mj *MultiTenantHandleT) getSingleWorkspaceQueryString(workspace string, count int, ds dataSetT, params GetQueryParamsT, order bool) string {
+func (mj *MultiTenantHandleT) getSingleWorkspaceQueryString(workspace string, count int, payloadLimit int64, ds dataSetT, params GetQueryParamsT, order bool) string {
```

Comment:
why do we need the `payloadLimit` parameter and how is this different from `params.PayloadSize`?

Reply:
`params.PayloadSize` as it is right now is the total payload limit. For all workspaces.
`payloadLimit` is just for that particular `workspace`

Reply:
The only reason we are passing `GetQueryParamsT` appears to be for including `params.StateFilters[0]` in a log message.  Can we exchange `params GetQueryParamsT` with `state string`?

Reply:
Good catch.
In fact I think we can leave `params` out entirely considering it's getting all jobs and including one state in the log doesn't add much.
We are setting it ourselves anyway(`params.StateFilters = []string{NotProcessed.State, Waiting.State, Failed.State}`)